changelog.txt

```diff
@@ -4,6 +4,7 @@
 * Fix - Refactor `WCS_Meta_Box_Subscription_Data::save` to support HPOS stores, fixing a PHP warning notice when updating an order via the Edit Order screen.
 * Update - Replace `get_post_type()` calls in `WC_Subscriptions_Manager` for HPOS support.
 * Update - Replace `get_post_type()` calls in `WC_Subscriptions_Synchroniser` for HPOS support.
+* Dev - Add subscriptions-core library version to the system status report.
```

Comment:
```suggestion
* Dev - Add subscriptions-core library version to the WooCommerce system status report.
```

Very minor suggestion: clarifying _which_ system status report (although I know there is probably only one). WooCommerce or WC?